### PR TITLE
Cyclic 1.12 Crafttweaker zenscript documentation

### PR DESCRIPTION
**Pull request by Lothrazar**
Based on 1.12 .  Some sample script files https://github.com/Lothrazar/Cyclic/tree/trunk/1.12/scripts_zen_example

### CODE REVIEW

**Comment by jaredlll08** [comment]
Don't forget to add the pages to the mkdocs.yml :)

**Comment by Lothrazar** [comment]
ok on it

File: docs/Mods/Cyclic/packager.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+Packager.removeRecipe(<minecraft:iron_ingot>);
+```
+
+
+#### Add Recipe
+
+```zenscript
+void addRecipe(
+  IItemStack output, // recipe result 
+  IIngredient input   // the single ingredient
```
Comment (kindlich) [inline docs/Mods/Cyclic/packager.md:29]: Your signature is `IItemStack`

File: docs/Mods/Cyclic/packager.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+);
+```
+
+```zenscript
+void addRecipe(
+  IItemStack output, //  recipe result
+  IIngredient[] input   // the ingredient list
```
Comment (kindlich) [inline docs/Mods/Cyclic/packager.md:36]: Your Signature is `IItemStack[]`

File: docs/Mods/Cyclic/solidifier.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+```
+
+
+#### Remove Recipes
+
+
+```zenscript
+void removeShapedRecipe(
+  string fluid, // the forge name of the fluid.  
+  int fluidSize
```
Comment (kindlich) [inline docs/Mods/Cyclic/solidifier.md:38]: Your Signature is `(IItemStack output)`

File: docs/Mods/Cyclic/solidifier.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+```
+
+
+#### Add Recipe
+
+```zenscript
+void addRecipe(
+  IItemStack output,   // the input as an itemstack
+  string fluid, // fluid name of the output
+  int fluidSize   // the number of fluidSize units to generate (1000 = 1 bucket)
```
Comment (kindlich) [inline docs/Mods/Cyclic/solidifier.md:27]: [Technical Question]
Why use String?
<liquid:water> * 1000 would be ILiquidStack and the name would be validated at syntax checks ^^

Comment (kindlich) [inline docs/Mods/Cyclic/solidifier.md:27]: fluidSize units?
Don't people normally use `mB` as in `milliBuckets` ?

**Comment by jaredlll08** [comment]
What is the current state of this PR?

**Comment by jaredlll08** [comment]
Closing due to inactivity.

**Pull request closed without merging**